gtests/test_nopython_radixtree.cpp

```diff
@@ -1,10 +1,37 @@
-#include <modmesh/toggle/RadixTree.hpp>
 #include <gtest/gtest.h>
+#include <modmesh/toggle/RadixTree.hpp>
 
 #ifdef Py_PYTHON_H
 #error "Python.h should not be included."
 #endif
 
+/**
+ * Simple Timed Entry for Testing
+ */
+class TimedEntry
```

Comment:
Moving the test code here makes sense to me.  What do you think, @q40603 ?

We have another `TimedEntry` at https://github.com/solvcon/modmesh/blob/d6ed9ac95f449ff77eb6e4d379c83d25f60eac23/cpp/modmesh/toggle/profile.hpp#L102 

For the test code here, what about renaming it to be different from `modmesh::TimedEntry` there?